Ansible/group_vars/kenet.yml

```diff
@@ -0,0 +1,161 @@
+---
+#My var just to be sure ansible is calling this file
+ronald: 1970233
+
+#Initialize generic with a value causing undefined in Ansible/roles/yaim/tasks/config_dirs.yml
+generic: generic
+
+#SkipCRL
+SkipCRLs: 'true'
+
+emi_service: UI
+needs_mysql: false
+yaim_options: '-n UI'
+yaim_path: /opt/glite/yaim
+yum_args: '--exclude="voms-clients-cpp"'
+bdii_port: 2170
+vo_sw_dir: /opt/exp_soft
+cvmfs_repos:
+  - apprepo.sagrid.ac.za
+  - atlas
+  - devrepo.sagrid.ac.za
+vos:
+  - dteam
+  - ops
+  - sagrid
+# - sagrid.ac.za
+  - eumed
+  - atlas
+queues:
+  - atlas
+  - dteam
+  - ops
+#  - sagrid.ac.za
+  - eumed
+# you can set VO-level queue tuning here
+qmgr_queues:
+  - name: dteam
+    resources_max.cput: '24:00:00'
+    max_wall_time: '30:00:00'
+  - name: ops
+    resources_max.cput: '0:15:00'
+    max_wall_time: '0:30:00'
+  - name: sagrid.ac.za
+    resources_max.cput: '72:00:00'
+    max_wall_time: '72:00:00'
+  - name: eumed
+    resources_max.cput: '72:00:00'
+    max_wall_time: '72:00:00'
+  - name: atlas
+    resources_max.cput: '72:00:00'
+    max_wall_time: '72:00:00'
+
+# See https://wiki.egi.eu/wiki/MAN01_How_to_publish_Site_Information
+# use the site name as registered in GOCDB (use the same case)
+site_name: ZA-MERAKA
+# contact details
+site_email: sagrid@csir.co.za
+site_security_email: csirt@sanren.ac.za
+site_support_email: za-meraka-ops@lists.sagrid.ac.za
+# location
+site_latitude: -25.75593
+site_longitude: 28.27921
+site_description: "Meraka Institute - core grid services"
+site_other_grid: "EGI"
+site_bdii_host: "site-bdii.c4.csir.co.za"
+site_location: "Pretoria, South Africa"
+site_website: "http://www.csir.co.za/meraka"
+site_domain: c4.csir.co.za
+
+# services at the site
+bdii_host: top-bdii.c4.csir.co.za
+apel_mysql_host: ce01-node001.c4.csir.co.za
+px_host: myproxy.cern.ch
```

Comment:
`px_host: myproxy.hellasgrid.gr`
This should be as above? Got it from https://wiki.egi.eu/wiki/Catch_All_Grid_Core_Services